src/ucp/core/ucp_worker.c

```diff
@@ -2939,14 +2953,14 @@ ucp_worker_do_keepalive_progress(ucp_worker_h worker)
         ucp_worker_keepalive_next_ep(worker);
     }
 
-    max_ka_ep_count = ucs_min(worker->context->config.ext.keepalive_num_eps,
-                              worker->num_all_eps) - worker->keepalive.ep_count;
+    max_ep_count = ucs_min(worker->context->config.ext.keepalive_num_eps,
```

Comment:
maybe rename 'ka_ep_count' to 'progress_count', to epmphasize it's used only for return value, and not for deciding when to stop keepalive loop?

Reply:
done